package.json

```diff
@@ -59,6 +59,7 @@
     "express-ipfilter": "^1.1.2",
     "express-mung": "^0.5.1",
     "isomorphic-fetch": "^3.0.0",
+    "memory-cache": "^0.2.0",
```

Comment:
https://github.com/vacuumlabs/adalite/blob/d1a9da4cfe87f0a6c6b1bbc50b93a029e493eb1e/app/frontend/wallet/blockchain-explorer.ts#L54

Isn't this simple helper we already have in our code do pretty much the same? If so, I'd rather not import new libraries if we don't really have to

Reply:
Fixed, I reused suggested code.